<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="sl_rotation_x">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sl_rotation_z">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_scale">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QSlider" name="sl_moving_x">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_moving_y">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MOVING</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_rotate_y">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="sl_rotation_y">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_moving_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="sl_scale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-500</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SCALE</string>
   </property>
  </widget>
  <widget class="QSlider" name="sl_moving_z">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_rotate_x">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_moving_x">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ROTATION</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_rotate_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="sl_moving_y">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-300</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
